### ISSUE TITLE
Transaction of HYDRA coin transfer cost more than 0.5 HYDRA and no refund is sent

### ISSUE THREAD

**Issue by codelano** [Hydra-Chain/node#7]
Please investigate why transaction of HYDRA coins transfer cost of 0.53822207 HYDRA
https://explorer.hydrachain.org/tx/d31a44e0d1b458278c7be8b0bdf41d430ce91d263a4513a9c52be7421827f227.
There are 0 pending HYDRA on the wallet

Wallet version: HYDRA version v0.18.5.4-f725a4441-dirty (64-bit)

![photo_2021-04-30_09-36-38](https://user-images.githubusercontent.com/15995240/116675111-211ec380-a9ae-11eb-9eca-62b2a76307ee.jpg)
![photo_2021-04-30_12-17-05](https://user-images.githubusercontent.com/15995240/116675113-21b75a00-a9ae-11eb-93e2-82f5eec5ce2c.jpg)

**Comment by Shiiike** [comment]
Hi, the exact same thing has happened to me today. I am using the same wallet version.

**Comment by codelano** [comment]
One interesting fact, if I restart wallet before transfer, fees are much lower, so I do encourage everyone before transferring to restart wallet software :), a quick workaround for this defect!

**Comment by Shiiike** [comment]
Okay, cheers.
Only problem is I had just updated my wallet before I made the transfer because one of the guys suggested I do that to make sure the right transfer fees were charged. Restarting might be different I guess.
Thanks for the email :)

    On Wednesday, 5 May 2021, 05:44:58 pm AEST, codelano ***@***.***> wrote:  
 
 


One interesting fact, if I restart wallet before transfer, fees are much lower, so I do encourage everyone before transferring to restart wallet software :), a quick workaround for this defect!
